CHANGELOG.md

```diff
@@ -1,9 +1,22 @@
-# dbt_amazon_ads_source v0.UPDATE.UPDATE
-
- ## Under the Hood:
-
+# dbt_amazon_ads_source v0.2.0
+## 🚨 Breaking changes
+- This release is labeled breaking to reflect the Fivetran Amazon Ads connector's upgrade from version 2 to 3 of the Sponsored Products API. Further details are also available in the [June 2023 release notes](https://fivetran.com/docs/applications/amazon-ads/changelog#june2023).
+## 🎉 Features
+[PR #11](https://github.com/fivetran/dbt_amazon_ads_source/pull/11) includes the following updates:
+- Added a new column, `native_language_locale`, to the `keyword_history` staging tables.
+- Made the following changes to the `campaign_history` table:
+  - Added three new columns, `budget`, `budget_type`, and `effective_budget`
+  - Removed the `campaign_type`, `daily_budget`, `placement`, and `premium_bid_adjustment` columns
+- Updated documentation with descriptions of the new columns.
+ ## 🚘 Under the Hood
+[PR #11](https://github.com/fivetran/dbt_amazon_ads_source/pull/11) includes the following updates:
```

Comment:
Since there is only one PR for these changes, you can most likely be comfortable putting this section right at the top of the CHANGELOG entry. This way you know all the below changes relate to this PR, and it doesn't need to be added in each individual section.

Reply:
There is also the package updater PR, but I think it looks ok this way now too. Updated.